pages/mesosphere/dcos/services/elastic/2.7.0-6.8.1/release-notes/index.md

```diff
@@ -0,0 +1,89 @@
+---
+layout: layout.pug
+navigationTitle:
+excerpt: Discover the new features, updates, and known limitations in this release of the Elastic Service
+title: Release Notes
+menuWeight: 10
+model: /mesosphere/dcos/services/elastic/data.yml
+render: mustache
+---
+
+# Release notes for Elastic Service version 2.7.0-6.8.1
+
+- [Upgrades](#orgdf76f77)
+- [New Features](#orgce810fb)
+- [Bug Fixes](#orgae15112)
+- [Improvements](#org630a806)
+
+
+<a id="orgdf76f77"></a>
+
+## Upgrades
+
+-   Upgraded Elasticsearch and Kibana to [6.8.1](https://www.elastic.co/guide/en/elasticsearch/reference/6.8/release-notes-6.8.1.html). This Elasticsearch version requires that nodes communicate through TLS if X-Pack Security is enabled. We made it so that when `elasticsearch.xpack_security_enabled` is set to `true` the scheduler will provision TLS artifacts and configure nodes to only accept TLS connections for both transport and HTTP mechanisms, similar to what one would get by setting `service.security.transport_encryption.enabled` to `true`. ([#12](https://github.com/mesosphere/dcos-elastic-service/pull/12))
+-   Upgraded SDK version from `0.55.4` to `0.56.2`.
+
+    Please take a look at the following SDK change logs for more information:
+
+    -   [0.55.5](https://github.com/mesosphere/dcos-commons/releases/tag/0.55.5)
+    -   [0.56.0](https://github.com/mesosphere/dcos-commons/releases/tag/0.56.0)
+    -   [0.56.1](https://github.com/mesosphere/dcos-commons/releases/tag/0.56.1)
+    -   [0.56.2](https://github.com/mesosphere/dcos-commons/releases/tag/0.56.2)
+
+    Changes: ([#34](https://github.com/mesosphere/dcos-elastic-service/pull/34))
+-   Replaced Oracle JRE (jre-8u192) with newer OpenJDK (openjdk-jre-8u212b03). ([#3060](https://github.com/mesosphere/dcos-commons/pull/3060))
+
+
+<a id="orgce810fb"></a>
+
+## New Features
+
+-   Added support for Elasticsearch plugin installation via proxy. New settings are:
+
+    -   `elasticsearch.plugin_http_proxy_host`
+    -   `elasticsearch.plugin_http_proxy_port`
+    -   `elasticsearch.plugin_https_proxy_host`
+    -   `elasticsearch.plugin_https_proxy_port`
+
+    Changes: ([#4](https://github.com/mesosphere/dcos-elastic-service/pull/4))
+-   Added custom domain support for Elastic. New setting is `service.security.custom_domain`. This custom domain is used in place of the default `autoip.dcos.thisdcos.directory`. This can be used to expose the service securely outside of the cluster, but requires setting up external DNS. Check [this documentation page](https://mesosphere.github.io/dcos-commons/developer-guide/#externalizing-transport-encryption-and-security) for more information. ([#9](https://github.com/mesosphere/dcos-elastic-service/pull/9))
```

Comment:
Same comment regarding adding a section for custom domains. Similar to [this](https://docs.d2iq.com/mesosphere/dcos/services/cassandra/2.6.0-3.11.4/security/#forwarding-dns-and-custom-domain)